File: shop/src/main/java/com/example/shop/dtos/response/ProductResponse.java
package com.example.shop.dtos.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ProductResponse {
    private String id;
    private String nameOfProduct;
    private String description;
    private Integer price;
    private Integer amount;
    private String picture;
}

File: Android app/app/src/main/java/com/example/shop/data/dtos/response/ProductResponse.kt
package com.example.shop.data.dtos.response

import lombok.AllArgsConstructor
import lombok.Data
import lombok.NoArgsConstructor



data class ProductResponse (
    val id: String,
    val nameOfProduct: String,
    val description: String,
    val price: Int,
    val amount: Int,
    val picture: String,
)